mp/src/game/client/momentum/ui/HUD/hud_timer.cpp

```diff
@@ -273,6 +274,20 @@ void CHudTimer::OnThink()
                     }
                 }
             }
+            else if (m_pRunData->m_iTimerState == TIMER_STATE_PRACTICE)
+            {
+                char curTime[BUFSIZETIME];
+                MomUtil::FormatTime(pEnt->GetCurrentRunTime(), curTime, 2);
+                m_pMainStatusLabel->SetText(curTime);
+
+                if (m_bWasUsingSavelocMenu)
+                    m_pInfoLabel->SetText(CConstructLocalizedString(m_wSavelocStatus, m_iSavelocCurrent, m_iSavelocCount));
+                else
+                    m_pInfoLabel->SetText("");
```

Comment:
Maybe clean this up with a ternary? That's the only thing I noticed in this PR.
```suggestion
                m_pInfoLabel->SetText(m_bWasUsingSavelocMenu ? CConstructLocalizedString(m_wSavelocStatus, m_iSavelocCurrent, m_iSavelocCount) : "");
```